spring-cloud-task-app-starters-docs/src/main/asciidoc/overview.adoc

```diff
@@ -1,7 +1,7 @@
 [[overview]]
-== Spring Cloud Stream Modules 
+== Spring Cloud Stream Starters
```

Comment:
@sobychacko when you merge this, please also change the term from `stream` to `task`